### ISSUE TITLE
On mastodon follow/unfollow: Unable to make activity POST: failed to decode PEM block containing private key

### ISSUE THREAD

**Issue by mrvdb** [writefreely/writefreely#602]
When issuing a follow/unfollow action on mastdon for /some/ writefreely blogs on our instance (qua.name) the log shows:

```
ERROR: 2022/11/22 09:45:35 activitypub.go:438: Unable to make activity POST: failed to decode PEM block containing private key
```
And the follow will not work for that particular blog. Other blogs *do* work.

Found one thread with the same problem, but no solution here: 
  https://mastodon.social/@evonity/108780888262381806

instance config: multiuser/mysql/invite only/federation enabled/v0.13.1

**Comment by mrvdb** [comment]
Ah: seem this bug https://github.com/writeas/web-core/issues/14

**Comment by mrvdb** [comment]
Resolved by installing 0.13.2 release.
